docs/DeveloperGuide.md

```diff
@@ -605,627 +629,630 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+Use case resumes from step 2.
 
-Preconditions: The students of the class have been added to TutorAid.
+* 1b. The list is empty.
 
-**MSS**
+Use case ends.
 
-1. Tutor requests to add a class
+* 3a. TutorAid detects that the command keyed in is incorrect. (e.g. wrong format, missing arguments, invalid indices)
+    * 3a1. TutorAid displays an error message and requests the tutor to re-enter the command.
+    * 3a2. Tutor re-enters the command, along with the necessary arguments.
 
-2. TutorAid requests for details of the class including the day of the week, time and price
+Steps 3a1-3a2 are repeated until the command entered is correct.
 
-3. Tutor enters the details of the class
+Use case resumes from step 4.
 
-4. TutorAid creates the class
+**Use case 15: Update payment status for a student**
 
-5. Tutor <u>adds a student to the class (UC14)</u>
+Preconditions: There is at least one student added to TutorAid.
 
-   Step 5 is repeated until all students have been added
+**MSS**
 
-   Use case ends.
+1. Tutor requests to view the list of their students.
+2. TutorAid shows the list of all students.
+3. Tutor identifies the index of the specific student for which the payment is to be updated to 'paid' or 'unpaid' and requests to update the payment status for this student accordingly.
+4. TutorAid successfully updates the payment status for the student corresponding to the given index number and displays a message to indicate this.
+   
+Use case ends.
 
 **Extensions**
 
-* 3a. Class appears to overlap with another class as intended by the Tutor
-
-    * 3a1. TutorAid shows a warning message
-
-    * 3a2. Tutor dismisses the warning message
-
-      Use case resumes at step 4
-
-* 3b. Class appears to overlap with another class due to a mistake of the Tutor
+* 1a. TutorAid detects an error in the list command.
+    * 1a1. TutorAid displays an error message and requests the tutor to re-enter the command.
+    * 1a2. User re-enters the command.
 
-    * 3b1. TutorAid shows a warning message
+Steps 1a1-1a2 are repeated until the command entered is correct.
 
-    * 3b2. Tutor acknowledges the warning message and cancels the creation of the class
+Use case resumes from step 2.
 
-      Use case resumes at step 1
+* 1b. The list is empty.
 
-* 3c. TutorAid detects invalid input
+Use case ends.
 
-    * 3c1. TutorAid shows a warning message
+* 3a. TutorAid detects that the command keyed in is incorrect. (e.g. wrong format, missing arguments, invalid indices)
+    * 3a1. TutorAid displays an error message and requests the tutor to re-enter the command.
+    * 3a2. Tutor re-enters the command, along with the necessary arguments.
 
-    * 3c2. Tutor acknowledges the warning message
+Steps 3a1-3a2 are repeated until the command entered is correct.
 
-      Use case resumes at step 3
+Use case resumes from step 4.
+      Use case resumes from step 5.
 
-**Use case 14: Add a student to a class**
+**Use case 16: Add multiple students to multiple lessons**
```

Comment:
If it is easier you can let me know what changes you would like to make and I can add it in? That way I thought it might be easier to format and merge later on 😅 But if you would like to make the change that is okay as well!

Reply:
If it is easier you can let me know what changes you would like to make and I can add it in? That way I thought it might be easier to format and merge later on 😅 But if you would like to make the change that is okay as well!

Reply:
Since we still allow adding multiple students to lessons, maybe we just need to update the warning. So we can still add if there are students already in class but there will be a warning.